src/main/java/Cart.java

```diff
@@ -0,0 +1,74 @@
+package main.java;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cart {
+    private final List<CartItem> cartItems; // избрани продукти
+    private double deliveryFee;
+
+    public Cart() {
+        this.cartItems = new ArrayList<>();
+    }
+
+    double getVAT() {
+        return this.calculateProductsPrices() * 0.2;
+    }
+
+
+    public double calculateFinalPrice() { //изчисляване на крайна цена на продуктите в кошницата заедно с ДДСто
+        double sumWithVAT = this.calculateProductsPrices() + getVAT();
+
+        if (this.calculateProductsPrices() < 100) {
+            this.deliveryFee = 10;
+        } else if (this.calculateProductsPrices() > 100 && this.calculateProductsPrices() < 200) {
+            this.deliveryFee = 5;
+        } else {
+            this.deliveryFee = 0;
+        }
+
+        return sumWithVAT + deliveryFee;
+    }
+
+    public void addItemsToCart(CartItem cartItem) {
+        this.cartItems.add(cartItem);
+    }
+
+    public void removeItemsFromCart(CartItem cartItem) {
```

Comment:
We should decouple the business objects from certain specifics as printing out stuff in the console.
This way you make the Cart class aware of the existence of the console